Doc/library/unittest.rst

```diff
@@ -220,6 +220,25 @@ Command-line options
 
    See `Signal Handling`_ for the functions that provide this functionality.
 
+.. cmdoption:: --debug
+
+   On test fail or error the test run terminates immediately with original
+   exception - similar to normal script execution. Useful for seamless external
+   post-mortem handling.
+
+.. cmdoption:: --pdb
+
+   Runs :func:`pdb.post_mortem` upon each error. Short for ``--pm=pdb``
+
+.. cmdoption:: --pm=<debugger>
+
+   Run custom post-mortem debugger (module or class) upon each error.
+   Examples: ``--pm=pywin.debugger``, ``--pm=IPython.terminal.debugger.Pdb``
```

Comment:
If the value is a module, what is actually called?

Reply:
The [# Simplified interface](https://github.com/python/cpython/blob/9db1e17c80217d7b18a2f44297d72acd10ee73d6/Lib/pdb.py#L1674) short-cuts ( `post_mortem()`, `runcall()` so far )

Reply:
Great, can you add it to the doc?

Reply:
So I'd change 
```diff
-   Run custom post-mortem debugger (module or class) upon each error.
+   Run custom post-mortem debugger (module or class) upon each error using its
+   ``post_mortem()`` function or method.
    Examples: ``--pm=pywin.debugger``, ``--pm=IPython.terminal.debugger.Pdb``
 
 .. cmdoption:: --trace
 
    Break at the beginning of each test using :mod:`pdb` or the debugger set by ``--pm``
+   via its ``runcall()`` function or method.

```

Reply:
Done